src/components/LibrariesPage.tsx

```diff
@@ -102,8 +102,7 @@ export class LibrariesPage extends React.Component<LibrariesPageProps, Libraries
   }
 
   /**
-  * setFilter()
-  * @param {string} filter - the library attribute to be filtered by
+  * @param {string} filter The library attribute to be filtered by
```

Comment:
Technically, `{type} nameOfParam` is not needed here because `Typedoc` takes care of figuring out the name and type of the param, but it doesn't hurt to have.